console/jsre/deps/doc.go

```diff
@@ -18,5 +18,18 @@
 // This file is derived from internal/jsre/deps/deps.go (2018/06/04).
 // Modified and improved for the klaytn development.
 
-// Package deps contains the console JavaScript dependencies Go embedded.
+/*
+Package deps contains the console's dependencies of Javascript libraries which is to be embedded in Go source code.
```

Comment:
```suggestion
Package deps contains the console's dependencies of Javascript libraries to be embedded in Go source code.
```
or "which is" -> "which are"